src/f32-vbinary/gen/f32-vmax-hvx-u64.c

```diff
@@ -0,0 +1,64 @@
+// Auto-generated file. Do not edit!
+//   Template: src/f32-vbinary/vop-hvx.c.in
+//   Generator: tools/xngen
+//
+
+#include <assert.h>
+
+#include <hvx_hexagon_protos.h>
+#include <hexagon_protos.h>
+#include <hexagon_types.h>
+
+#include <xnnpack/common.h>
+#include <xnnpack/intrinsics-polyfill.h>
+#include <xnnpack/math.h>
+#include <xnnpack/vbinary.h>
+
+void xnn_f32_vmax_ukernel__hvx_u64(
+    size_t batch,
+    const float* input_a,
+    const float* input_b,
+    float* output,
+    const union xnn_f32_default_params params[restrict XNN_MIN_ELEMENTS(1)]) XNN_OOB_READS
+{
+  assert(batch != 0);
+  assert(batch % sizeof(float) == 0);
+  assert(input_a != NULL);
+  assert(input_b != NULL);
+  assert(output != NULL);
+
+
+  const HVX_UVector *vptr_a = (const HVX_UVector *) input_a;
```

Comment:
UVector is unaligned?  Does it hurt performance much?

Reply:
> UVector is unaligned? Does it hurt performance much?

UVector is aligned with 4 Byte, and I actually found the performance got better for some cases with simulator (e.g., when we have the number of elements that fills up 128 bytes completely). We will find more accurate performance later with actual device.